django_typeform/responses.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8
+"""
+Implement the response API of typeform_id.
+
+The class TypeformResponse reads JSON objects as described on the typeform_id developer web site:
+https://developer.typeform.com/
+"""
+
+import json
+from urllib.error import HTTPError
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
+
+from dateutil.parser import parse
+from django.conf import settings
+from django.http import QueryDict
+
+TYPEFORM_SERVER = getattr(settings, 'TYPEFORM_SERVER', 'https://api.typeform.com/')
+if TYPEFORM_SERVER[-1] != '/':  #  add trailing slash if missing
+    TYPEFORM_SERVER += '/'
+TYPEFORM_FORM = getattr(settings, 'TYPEFORM_FORM', None)
+TYPEFORM_TOKEN = getattr(settings, 'TYPEFORM_TOKEN', "")
+TYPEFORM_CHOICES = getattr(settings, 'TYPEFORM_CHOICES', {})
+
+class TypeformResponses(object):
+    form = None
+
+    decode = {
```

Comment:
Not quite following how `decode` is being used. Could you explain a little bit how and where it's being used

Reply:
Did do so. Hope it helps.